index.html

```diff
@@ -353,21 +353,21 @@ <h6 id="skills-title">Skills</h6>
         </div>
       </div> 
     </section>
-    <section class="contact-form-container">
+    <footer class="contact-form-container">
         <div class="description">
           <h6 class="description-text">I'm always interested in hearing about new projects,
              so if you'd like to chat please get in touch
           </h6>
         </div>
         <div class="contact-form">
-          <form action="#" id="form">
-            <input type="text" name="fullname" id="fullname" placeholder="Full name">
-            <input type="email" name="email" id="email" placeholder="Email address">
-            <textarea name="textarea" id="textarea" cols="30" rows="10" placeholder="Enter text here"></textarea>
+          <form action="https://for,spree.io/f/{form_id}" id="form" method="post">
+            <input type="text" maxlength="30" name="fullname" id="fullname" placeholder="Full name" required>
```

Comment:
Kindly correct the action to use a valid [formspree service](https://formspree.io/html) URL so that forms can be properly submitted without errors.

Response:
![image](https://user-images.githubusercontent.com/20554583/193379070-2508b105-7aa5-45fe-9364-3722d6d37411.png)